pkg/generators/forwarding/fluentd/generators.go

```diff
@@ -14,9 +15,10 @@ import (
 //ConfigGenerator is a config generator for fluentd
 type ConfigGenerator struct {
 	*generators.Generator
-	includeLegacyForwardConfig bool
-	includeLegacySyslogConfig  bool
-	useOldRemoteSyslogPlugin   bool
+	includeLegacyForwardConfig    bool
```

Comment:
I hope we can refactor this like the GA API. Using a common part and a type specific part. Otherwise this gets overcrowded. For syslog there will be few more things added here.

Reply:
These are deprecated features that will eventually go away and be replaced by the new-API equivalent. We don't want to spend time on them, but if the clutter gets in the way of developing new features it might be worth refactoring as you suggest.